### PR TITLE
Refactor coercers

### PR DESCRIPTION
**Pull request by shosca**
Make'em more django-like

### CODE REVIEW

File: django_sorcery/db/meta/column.py [hunk 1]
```diff
@@ -156,6 +166,11 @@ def field_kwargs(self):
         kwargs["max_length"] = self.column.type.length
         return kwargs
 
+    def to_python(self, value):
+        if value is None:
+            return value
+        return str(value).strip()
```
Comment (miki725) [inline django_sorcery/db/meta/column.py]: why not leverage generated form field? seems like duplication

Comment (shosca) [reply django_sorcery/db/meta/column.py]: Yes it seems so, even in django itself there's some duplication, as in form field `to_python` and db field `to_python`. But i don't like the idea to initialize a form field just to be able to coerce data types. 

As a side note, i'm thinking about adding a `clean` here and move the model field cleaning here as well.

Comment (miki725) [reply django_sorcery/db/meta/column.py]: You don't need to initialize complete form. Just call directly fields clean.

Comment (shosca) [reply django_sorcery/db/meta/column.py]: added a `coercer` that can be a formfield instance that's lazily initialized

**Pull request merged**